salt/lax/bot-lax-adaptor.sls

```diff
@@ -149,22 +150,30 @@ bot-lax-uwsgi-conf:
             - bot-lax-adaptor-install
             - bot-lax-writable-dirs
 
-{% set apiprotocol = 'https' if salt['elife.cfg']('cfn.outputs.DomainName') else 'http' %}
-{% set apihost = salt['elife.cfg']('project.full_hostname', 'localhost') %}
-
-uwsgi-bot-lax-adaptor:
+bot-lax-uwsgi-upstart:
     file.managed:
         - name: /etc/init/uwsgi-bot-lax-adaptor.conf
         - source: salt://lax/config/etc-init-uwsgi-bot-lax-adaptor.conf
         - template: jinja
         - mode: 755
 
+bot-lax-uwsgi-systemd:
+    file.managed:
+        - name: /lib/systemd/system/uwsgi-bot-lax-adaptor.service
+        - source: salt://lax/config/lib-systemd-system-uwsgi-bot-lax-adaptor.service
+        - template: jinja
+        - mode: 644
```

Comment:
I would check what happens to https://github.com/elifesciences/lax-formula/blob/master/salt/lax/processes.sls#L14
On 14.04 it shouldn't be affected, it continues to run the old script for Upstart processes.
On 16.04 I don't know what it would do as it tries to start the other Upstart processes explicitly.